public/admin/config.yml

```diff
@@ -644,3 +644,24 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
+  - label: Vaccinations directory
+    name: vaccines
+    folder: 'public/cms-content/vaccines'
+    create: true
+    label_singular: Vaccine
+    description: description
+    format: json
+    identifier_field: state
+    fields:
+      - label: State
+        name: state
```

Comment:
Consider renaming the label and name to `State Code` and `stateCode` respectively